csunplugged/topics/models.py

```diff
@@ -45,6 +45,7 @@ class CurriculumArea(models.Model):
     #  Auto-incrementing 'id' field is automatically set by Django
     slug = models.SlugField(unique=True)
     name = models.CharField(max_length=100, unique=True)
+    colour = models.CharField(max_length=15, null=True)
```

Comment:
this should be longer. What if we eventually just use web colour codes rather than mapping the colours ourselves? https://en.wikipedia.org/wiki/Web_colors - several of those colours are longer than 15 characters long.